src/App.tsx

```diff
@@ -113,9 +113,9 @@ class App extends React.Component<RouteComponentProps, State> {
     this.props.history.push('/')
   }
 
-  public updateVote = (contestId: string, candidate: OptionalCandidate) => {
+  public updateVote = (contestId: string, vote: OptionalVote) => {
```

Comment:
If confusing, let's update the terminology. Please propose any terms you find less confusing.

Here's my current thinking…

It hasn't been defined how a voter's "ballot" will be distilled from an election, but I've been thinking that `this.state.ballot` would be used for this. 

Possible `App` state:

```
{
  election: Election // <-- election config json
  ballot: Ballot // <-- TBD, created when an activation code is entered.
  votes: Votes {
    contestId: Candidate[] || Approval // <-- Approval will be a boolean type
  }
}
```

If the voter's "ballot" is dynamically derived from the election (instead of creating a ballot when an activation code is entered) then we can change `votes` to `ballot`.